src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine{
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator stop work");
+    }
+
+    @Override
+    public void doWork() {
+        System.out.println("Excavator stop work");
```

Comment:
The message in `doWork` method is incorrect. It should print a message indicating that the excavator is starting work, such as `"Excavator start work"`.